<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchlistTrackDialogClass</class>
 <widget class="QDialog" name="SearchlistTrackDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Searchlist Edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/resources/edit.png</normaloff>:/icons/resources/edit.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Artist:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="artistEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="versionEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Genre:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="genreEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Length:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lengthEdit">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string notr="true">0009:99; </string>
         </property>
         <property name="text">
          <string notr="true">:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>min.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="commentEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Releasedate:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="releaseDateEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Bewertung:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rating">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>16</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/bewertung/resources/sterne-unbewertet.png</normaloff>:/bewertung/resources/sterne-unbewertet.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/bewertung/resources/sterne1.png</normaloff>:/bewertung/resources/sterne1.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/bewertung/resources/sterne2.png</normaloff>:/bewertung/resources/sterne2.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/bewertung/resources/sterne3.png</normaloff>:/bewertung/resources/sterne3.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/bewertung/resources/sterne4.png</normaloff>:/bewertung/resources/sterne4.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/bewertung/resources/sterne5.png</normaloff>:/bewertung/resources/sterne5.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/bewertung/resources/sterne6.png</normaloff>:/bewertung/resources/sterne6.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/resources/filesave.png</normaloff>:/icons/resources/filesave.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/resources/cancel.png</normaloff>:/icons/resources/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>artistEdit</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>versionEdit</tabstop>
  <tabstop>genreEdit</tabstop>
  <tabstop>lengthEdit</tabstop>
  <tabstop>commentEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
